pkg/arvo/sys/vane/ames.hoon

```diff
@@ -5886,33 +5915,15 @@
     |^
     =/  van  ?@(vis.nom (end 3 vis.nom) way.vis.nom)
     =/  kyr  ?@(vis.nom (rsh 3 vis.nom) car.vis.nom)
-    ?+    van  ~
-        %a
-      ?+  kyr  ~
-        %x  (en-hunk (rof lyc /ames nom))
-      ==
-    ::
-        %c
-      =+  pem=(rof lyc /ames nom(vis %cp))
-      ?.  ?=(^ pem)    ~
-      ?.  ?=(^ u.pem)  ~
-      ~|  u.u.pem
-      =+  per=!<([r=dict:clay w=dict:clay] q.u.u.pem)
-      ?.  =([%black ~ ~] rul.r.per)  ~
+    ?.  =(%c van)
       (en-hunk (rof ~ /ames nom))
-    ::
-        %e
-      %-  en-hunk
-      ?+  kyr  ~
-        %x  (rof ~ /ames nom)
-      ==
-    ::
-        %g
-      %-  en-hunk
-      ?+  kyr  ~
-        %x  (rof ~ /ames nom)
-      ==
-    ==
+    =+  pem=(rof [~ ~] /ames nom(vis %cp))
```

Comment:
Well almost all of them. Clay still doesn't respect the gang system properly. This shouldn't matter, but we should test thoroughly to make sure.